### PR TITLE
[DDIDO-594] Added Drush dependency.

### PR DESCRIPTION
**Pull request by GROwen**
### Changes
Moves the declaration of Drush dependency to this project from downstreams.
Updates the repository for section_purger to Drupal
Adds a patch to fix an error thrown when running Drush 10 updb

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,10 +4,11 @@
     "type": "drupal-module",
     "license": "GPL-2.0-or-later",
     "require": {
+        "drush/drush": "^10.3",
         "drupal/purge": "^3.0@beta",
         "drupal/purge_queuer_url": "^1.0@RC",
         "drupal/redis": "^1.0@RC",
-        "drupal/section_purger": "0.2.3",
+        "drupal/section_purge": "2.3-rc2",
```
Comment (anthony-malkoun) [inline composer.json:11]: Has the name of the module actually changed?

**Comment by nicksantamaria** [comment]
I'm impressed at the commit to line change ratio

File: composer.json (hunk 1 above)
Comment (nicksantamaria) [reply composer.json:11]: Just confirmed - yes it has.

- OK: https://www.drupal.org/project/section_purge
- 404: https://www.drupal.org/project/section_purger

**Comment by GROwen** [comment]
ha yes, once I found it was possible to use wildcard aliases...

https://images.app.goo.gl/AShhmAbVab8TwVPY6

**Comment by nicksantamaria** [comment]
Is there a corresponding build of content-vic or something using this branch?

**Comment by GROwen** [comment]
https://github.com/dpc-sdp/content-vic-gov-au/tree/release/1.32.2

**Comment by GROwen** [comment]
Naming was set after a run of the upgrade and conflicts with an actual release.

**Review by nicksantamaria — APPROVED** [review]

**Pull request merged**